variables.tf

```diff
@@ -377,6 +377,36 @@ variable "confluence_max_heap" {
   default     = "512m"
 }
 
+variable "synchrony_cpu" {
+  description = "Number of CPUs for synchrony instance"
+  type        = string
+  default     = "2"
+}
+
+variable "synchrony_mem" {
+  description = "Amount of memory for synchrony instance"
+  type        = string
+  default     = "2.5Gi"
+}
+
+variable "synchrony_min_heap" {
+  description = "Minimum heap size for synchrony instance"
+  type        = string
+  default     = "1Gi"
```

Comment:
It has be be `1g` otherwise JVM won't be created - https://github.com/atlassian-labs/data-center-terraform/actions/runs/3262312392/jobs/5359248528#step:11:10025

These are logs from synchrony pod:
```
Invalid initial heap size: -Xms1Gi
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```